### ISSUE TITLE
CSED'18 bot Feedback

### ISSUE THREAD

**Issue by basmataha199** [csed2018/messengerBot#7]
1) The table as written statements doesn't seem so good to me, as a viewer. I don't know how applicable the idea is, but I got one, maybe you can preview the table as an image with rows and columns.
2) Also in the description "The menu (the 3-caret icon on the left of the composer)", I didn't find such thing in that menu, if you meant what I understood (maybe it's my problem).

3) The help description is not, somehow, user-friendly. Maybe you can expose your idea more with symbols or icons.

4) I am not an English expert :D but I think there are some grammar mistakes.

5) Maybe you should write less text and make the interaction more with symbols and icons, cause sometimes reading long statements and paragraphs is boring. (and sorry for being boring, myself :3)

**Comment by HazemSamir** [comment]
I totally agree with all your points. I really like the idea of the table as an image, i'll try to implement it soon isA.

by the menu I mean:
![screenshot from 2017-03-01 00 16 47](https://cloud.githubusercontent.com/assets/12578145/23436790/7ab4452c-fe14-11e6-81b4-337d16f99bff.png)
It should appear on desktop browsers and messenger app.

Concerning the help text, it's the same thing I think about every single time I see it :sweat_smile: 

Sure, I never write a text that is grammatically correct, I expect mistakes all the time. I'll be happy if you report them.

Messenger provides many templates other than simple text messages. I'll try to use them to make it less "texty" and more user friendly.

Feel free to browse them yourself. If you get any idea, let me know https://developers.facebook.com/docs/messenger-platform/send-api-reference/templates

Thanks very much for your feedback :smiley:

**Comment by basmataha199** [comment]
I liked this button preview:
https://developers.facebook.com/docs/messenger-platform/send-api-reference/button-template

You may put the help as this list of buttons when user tells the bot "help" or "show" or whatever you like:
- Next day's schedule
- Tomorrow's Assignments
- This week's Assignments
... etc
_**I think you can preview all of this stuff in an image table, as well.**_

It'd be a lot better and self-descriptive. 
I found the menu, thank you. Maybe you can describe it using an image or small icon like you did now with me, to be more noticeable. 

You are very welcome, keep it up. It's a greatly useful bot :D

**Comment by HazemSamir** [comment]
Hey, check the help now. I'll be happy to get your feedback.

If you find any typo or a grammar mistake please let me know.

**Comment by basmataha199** [comment]
It's better than before, though I have one comment: I think the shortcut description isn't uniform for all the shortcuts. For example, the "24 hrs assignments" have 2 shortcuts while "7 days assignments" have four, and next day schedule has 4 so I think you better use one pattern for shortcuts. Besides, I prefer to use as fewer shortcuts as possible, cause I think anyone will only pick one and stick to it.

The one grammar mistake, I noticed:

"Take Care :v If the day I send you is different than the acutal tomorrow, then the schedule is not confirmed yet"

- **different from (not than)**

- **actual (there was a typo in the typed word)**
_If you want my opinion in this statement, I think for simplicity you might say: "Take care: If the schedule is not tomorrow's, then the schedule is not confirmed yet. "_ 

**BTW**, you can use this chrome extension for grammar checking in whatever text you type in chrome:
https://chrome.google.com/webstore/detail/grammarly-for-chrome/kbfnbcaeplbcioakkpcpgfkobkghlhen?utm_source=chrome-app-launcher-info-dialog

**Comment by basmataha199** [comment]
Oh, I forgot something, when I hit subscribe or unsubscribe, the bot replies by "Subscribe/Unsubscribe", but doesn't tell me What I (un)subscribed for. I think it'd be better if it tells something like: "You're now (un)subscribed (from)for bot features and updates", for example.
Same thing goes for "learn more".

I know those are small things, but I hope they might make a difference! :D

**Comment by basmataha199** [comment]
It's a lot better now. Thank you.

**Comment by HazemSamir** [comment]
Thank you so much for sharing your ideas and feedback. If you have any other thoughts, please don't hesitate to share them any time :smiley: 
I didn't add all things you suggested but I'll add some of them later :grinning:
